src/scripts/site.js

```diff
@@ -1106,7 +1135,8 @@ var Dashboard = {
             map: {
                 "*": {
                     css: bowerPath + "/emby-webcomponents/require/requirecss",
-                    text: bowerPath + "/emby-webcomponents/require/requiretext"
+                    text: bowerPath + "/emby-webcomponents/require/requiretext",
+                    //appStorage: getAppStorage(apiClientBowerPath)
```

Comment:
is this comment needed?

Reply:
No

Reply:
Can wee revert these two lines then? I don't see the point in changing them in such case.